scripts/generate-react-exports.js

```diff
@@ -0,0 +1,109 @@
+const fs = require('fs-extra');
+const prettier = require('prettier');
+
+function getComponentNameDict(baklavaFileContent) {
+  const componentNameDict = {};
+  const lines = baklavaFileContent.split(/\r\n|\n/).filter(line => line !== '');
+
+  for (const line of lines) {
+    try {
+      const {
+        groups: { componentName, fileName },
+      } = line.match(/as (?<componentName>.*) } from ['`"](?<path>.*\/(?<fileName>[\w-]*))['`"];$/);
+
+      componentNameDict[componentName] = fileName;
+    } catch (error) {}
+  }
+
+  return componentNameDict;
+}
```

Comment:
We generate `custom-elements.json` file and put to dist folder with `npm run analyze` command. We can use that json file to read component list and their properties etc. I think this can simplify process here a lot.

Reply:
We can inspire by Shoelace's react component wrapper generator: https://github.com/shoelace-style/shoelace/blob/next/scripts/make-react.js

Reply:
we didn't want to use custom-elements.json to be included in the build. then we can run `npm run analyze` before the generate step.